src/komponenter/banner/banner.tsx

```diff
@@ -37,8 +37,7 @@ class Banner extends React.Component<Props> {
 
     skalVises(): boolean {
         return !(document.location.pathname.includes(START_PATH)
-            && (this.props.startRegistreringStatus.underOppfolging === false));
```

Comment:
skal ikke denne vises for f.eks sykemeldt-registrering også ? tror kanskje vi må bevare "underOppfolging".

Reply:
Setter tilbake